src/edgepi/peripherals/pwm.py

```diff
@@ -74,33 +75,43 @@ def set_duty_cycle_pwm(self, duty_cycle: float = None):
 
     def get_duty_cycle_pwm(self):
         """
-        get pwm duty cycle
+        Get pwm duty cycle
         Args:
             N/A
         Return:
             duty_cycle (float): duty cycle value from 0 to 1.0
         """
         return self.pwm.duty_cycle
 
-    def set_polarity_pwm(self, polarity: str = None):
+    def set_polarity_pwm(self, polarity: Polarity = None):
         """
-        set pwm polarity
+        Set pwm polarity
         Args:
-            polarity (str): "Normal" or "Inversed"
+            polarity (Polarity)
         Return:
             N/A
         """
-        self.pwm.polarity = polarity
+        if polarity == Polarity.NORMAL:
+            self.pwm.polarity = "normal"
+        elif polarity == Polarity.INVERSED:
+            self.pwm.polarity = "inversed"
+        else:
+            raise ValueError(f"{polarity} is not a valid value for polarity.")
 
     def get_polarity_pwm(self):
         """
-        get pwm polarity
+        Get pwm polarity
         Args:
             N/A
         Return:
-            polarity (str): "Normal" or "Inversed"
+            polarity (Polarity)
         """
-        return self.pwm.polarity
+        if self.pwm.polarity == Polarity.NORMAL.name.lower():
+            return Polarity.NORMAL
+        if self.pwm.polarity == Polarity.INVERSED.name.lower():
+            return Polarity.INVERSED
```

Comment:
I meant use literal strings to compare to rather than relying on enum names.

Reply:
Done. https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/330/commits/7c263383dca4c0efccc179ba958ad92ea60d00ea